platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -485,11 +485,19 @@ data:
   #
   #
   type: h2
+
   ## @param: url
   ## @type: string
   ## Location of the database
   #
-  url: jdbc:h2:mem:anchor-platform
+#  url: jdbc:h2:mem:anchor-platform
+
+  ## The hostname of the database server.
+  server:
+
+  ## Name of the database. Optional. Default to anchor-platform or the default name of the type of database.
```

Comment:
Does this mean in postgres, it defaults to "postgres"? What do you think about requiring this field instead?

Reply:
I think we should make it required. If the database does not exist, AP will fail. 

We can add automatic database creation if that is becoming an issue.

Reply:
Awesome, lets do it.